package.json

```diff
@@ -19,15 +20,24 @@
   },
   "homepage": "https://github.com/anthony-raybould/bham1-front-end#readme",
   "dependencies": {
```

Comment:
Dependencies need looking at and splitting into needed for application code (dependency) or needed for dev (dev dependency)